### PR TITLE
Create WonahDshypoet.json

### PR DESCRIPTION
**Pull request by WonahDshypoet**
## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [x] My change requires changes to the documentation.
- [x] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

### CODE REVIEW

File: data/WonahDshypoet.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+      "name": "X",
+      "url": "X.com/shypoet__",
+      "icon": "FaTwitter",
+    },
+    {
+      "group": "socail",
+      "name": "Linkedin"
+      "url": "https://www.linkedin.com/in/godwin-wonah-9021371ab/",
+      "icon": "FaLinkedin",
+    },
```
Comment (sital002) [inline data/WonahDshypoet.json:16]:
```suggestion
    }
```
The trailing comma isnot required.

File: data/WonahDshypoet.json [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+      "group": "social",
+      "name": "X",
+      "url": "X.com/shypoet__",
+      "icon": "FaTwitter",
+    },
+    {
+      "group": "socail",
+      "name": "Linkedin"
+      "url": "https://www.linkedin.com/in/godwin-wonah-9021371ab/",
+      "icon": "FaLinkedin",
```
Comment (sital002) [inline data/WonahDshypoet.json:15]:
```suggestion
      "icon": "FaLinkedin"
```
The trailing comma isnot required

File: data/WonahDshypoet.json [hunk 3]
```diff
@@ -0,0 +1,18 @@
+{
+  "name": "Godwin Wonah",
+  "bio": "Software Developer",
+  "links": [
+    {
+      "group": "social",
+      "name": "X",
+      "url": "X.com/shypoet__",
+      "icon": "FaTwitter",
```
Comment (sital002) [inline data/WonahDshypoet.json:9]:
```suggestion
      "icon": "FaTwitter"
```

**Review by sital002 — CHANGES_REQUESTED** [review]
Please make the necessary changes to get your profile merged

File: data/WonahDshypoet.json (hunk 1 above)
Comment (WonahDshypoet) [reply data/WonahDshypoet.json:14]: Okay.

**Review by sital002 — APPROVED** [review]
Looks good 👍
Welcome to Eddiehub community 🎉🎉

**Pull request merged**